test/e2e/adminapi_kubernetesobjects.go

```diff
@@ -0,0 +1,350 @@
+		ctx = context.Background()
+		resourceID = resourceIDFromEnv()
+		namespace = "default"
+		objName = "e2e-test-object"
+	})
+
+	testSecretOperationsForbidden := func() {
+		It("should not be able to create a secret", func() {
+			By("creating a new secret via RP admin API")
+			body := fmt.Sprintf(
+				`{
+					"kind": "Secret",
+					"apiVersion": "v1",
+					"metadata": {
+						"name": "%s",
+						"namespace": "%s"
+					},
+					"type": "Opaque",
+					"data": {
+						"password": "MWYyZDFlMmU2N2Rm"
+					}
+				}`,
+				objName,
+				namespace,
+			)
+
+			var cloudErr api.CloudError
+			resp, err := adminRequest(ctx, resourceID, http.MethodPost, "/kubernetesobjects", nil, json.RawMessage(body), &cloudErr)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.StatusCode).To(Equal(http.StatusForbidden))
+			Expect(cloudErr.Code).To(Equal(api.CloudErrorCodeForbidden))
+		})
+
+		It("should not be able to get a secret", func() {
+			By("requesting a secret via RP admin API")
+			params := url.Values{
+				"kind":      []string{"secret"},
+				"namespace": []string{namespace},
+				"name":      []string{objName},
+			}
+			var cloudErr api.CloudError
+			resp, err := adminRequest(ctx, resourceID, http.MethodGet, "/kubernetesobjects", params, nil, &cloudErr)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.StatusCode).To(Equal(http.StatusForbidden))
+
+			By("checking response for error")
+			Expect(cloudErr.Code).To(Equal(api.CloudErrorCodeForbidden))
+		})
+
+		It("should not be able to get a list of secrets", func() {
+			By("requesting a list of Secret objects via RP admin API")
+			params := url.Values{
+				"kind":      []string{"secret"},
+				"namespace": []string{namespace},
+			}
+			var cloudErr api.CloudError
+			resp, err := adminRequest(ctx, resourceID, http.MethodGet, "/kubernetesobjects", params, nil, &cloudErr)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.StatusCode).To(Equal(http.StatusForbidden))
+
+			By("checking response for error")
+			Expect(cloudErr.Code).To(Equal(api.CloudErrorCodeForbidden))
+		})
+
+		It("should not be able to delete a secret", func() {
+			By("deleting the secret via RP admin API")
+			params := url.Values{
+				"kind":      []string{"secret"},
+				"namespace": []string{namespace},
+				"name":      []string{objName},
+			}
+			var cloudErr api.CloudError
+			resp, err := adminRequest(ctx, resourceID, http.MethodDelete, "/kubernetesobjects", params, nil, &cloudErr)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.StatusCode).To(Equal(http.StatusForbidden))
+			Expect(cloudErr.Code).To(Equal(api.CloudErrorCodeForbidden))
+		})
+	}
+
+	testConfigMapCreateOK := func() {
+		By("creating a new object via RP admin API")
+		body := fmt.Sprintf(
+			`{
+				"kind": "ConfigMap",
+				"apiVersion": "v1",
+				"metadata": {
+					"name": "%s",
+					"namespace": "%s"
+				},
+				"data": {
+					"key": "value"
+				}
+			}`,
+			objName,
+			namespace,
+		)
```

Comment:
this same configmap is being defined 3 times in this source file.  This isn't mergeable.